avm/res/databricks/workspace/tests/e2e/max/main.test.bicep

```diff
@@ -12,7 +12,7 @@ metadata description = 'This instance deploys the module with most of its featur
 param resourceGroupName string = 'dep-${namePrefix}-databricks.workspaces-${serviceShort}-rg'
 
 @description('Optional. The location to deploy resources to.')
-param location string = deployment().location
+param location string = 'westeurope'
```

Comment:
The change in this and the other files should be undone as it's anyways overwritten

Reply:
@ChrisSidebotham, @AlexanderSehr maybe I'm missing something but how is one supposed to set a fixed location for his test deployments to avoid deployment errors due to service unavailability in certain minor regions?

Reply:
The best model I've seen used by e.g. @fabmas was to create a variable

https://github.com/Azure/bicep-registry-modules/blob/314e95f478b75f5fadb25b0b16c33a7855b5c16d/avm/res/network/virtual-network-gateway/tests/e2e/defaults/main.test.bicep#L24

and reference that in the dependencies & main deployment.

The problem overwriting the default is just that - it's a default :D  We do pass an explicit value in and hence the default isn't used. @ChrisSidebotham  already started working on a location rotation solution, so hopefully in the mid-term we could remove the hardcoded locations again.